core/sdk/src/api/firebolt/fb_metrics.rs

```diff
@@ -391,6 +394,195 @@ impl BehavioralMetricPayload {
     }
 }
 
+/*
+Operational Metrics. These are metrics that are not directly related to the user's behavior. They are
+more related to the operation of the platform itself. These metrics are not sent to the BI system, and
+are only used for operational/performance measurement - timers, counters, etc -all of low cardinality.
+*/
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Clone)]
+pub struct Counter {
```

Comment:
@bobra200 : From your POC. Did we decide to keep `Counter` in for now, or remove and add later if needed?

Reply:
I think it's keep so far, think we just don't know we need it yet